packages/cli/fixtures/dom-parser-test.tsx

```diff
@@ -0,0 +1,318 @@
+import {
+  h,
+  type Cell,
+  cell,
+  derive,
+  handler,
+  ID,
+  type JSONSchema,
+  NAME,
+  recipe,
+  type Schema,
+  str,
+  UI,
+} from "commontools";
+import { DOMParser, type Element } from "dom-parser";
```

Comment:
This and the gmail recipe in fixtures could be significantly stripped down if just testing 3p libs